src/commands/createFunction/openAPISteps/OpenAPICreateStep.ts

```diff
@@ -29,46 +30,68 @@ export class OpenAPICreateStep extends AzureWizardExecuteStep<IFunctionWizardCon
             case ProjectLanguage.TypeScript:
-                args.push('--azure-functions-typescript');
+                pluginRun = 'typescript';
                 args.push('--no-namespace-folders:True');
                 break;
             case ProjectLanguage.CSharp:
+                pluginRun = 'csharp';
                 args.push(`--namespace:${nonNullProp(wizardContext, 'namespace')}`);
-                args.push('--azure-functions-csharp');
                 break;
             case ProjectLanguage.Java:
+                pluginRun = 'java';
                 args.push(`--namespace:${nonNullProp(wizardContext, 'javaPackageName')}`);
-                args.push('--azure-functions-java');
                 break;
             case ProjectLanguage.Python:
-                args.push('--azure-functions-python');
-                args.push('--no-namespace-folders:True');
+                pluginRun = 'python';
                 args.push('--no-async');
+                args.push('--no-namespace-folders:True');
                 break;
             default:
                 throw new Error(localize('notSupported', 'Not a supported language. We currently support C#, Java, Python, and Typescript'));
-                break;
         }
 
-        args.push('--generate-metadata:false');
+        args.push(`--${pluginRunPrefix}-${pluginRun}`);
+        args.push(`--input-file:${cpUtils.wrapArgInQuotes(uri.fsPath)}`);
         args.push(`--output-folder:${cpUtils.wrapArgInQuotes(wizardContext.projectPath)}`);
+        args.push('--generate-metadata:false');
+        args.push(`--version:3.0.6320`);
 
         ext.outputChannel.appendLog(localize('statutoryWarning', 'Using the plugin could overwrite your custom changes to the functions.\nIf autorest fails, you can run the script on your command-line, or try resetting autorest (autorest --reset) and try again.'));
-        const title: string = localize('generatingFunctions', 'Generating from OpenAPI...Check [output window](command:{0}) for status.', ext.prefix + '.showOutputChannel');
+        ext.outputChannel.appendLog(localize('urlsForIssues', 'If there are any issues, please visit https://aka.ms/stencil for usage information or report issues at https://aka.ms/stencil/issues.'));
+        const title: string = localize('generatingFunctions', 'Generating from OpenAPI Specification...Check [output window](command:{0}) for status.', ext.prefix + '.showOutputChannel');
 
         await window.withProgress({ location: ProgressLocation.Notification, title }, async () => {
-            await cpUtils.executeCommand(ext.outputChannel, undefined, 'autorest', ...args);
+            try {
+                await cpUtils.executeCommand(ext.outputChannel, undefined, 'autorest', ...args);
+            } catch (error) {
+                const message: string = localize('autorestNotRunSuccessfully', `Failed to run "autorest.${pluginRunPrefix}-${pluginRun}" | Click "Report an Issue" to report the error.`);
```

Comment:
Please remove `| Click "Report an Issue" to report the error.`. We try to keep messages short, and it's pretty redundant with the button itself.